src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedDeletedMethod.cs

```diff
@@ -0,0 +1,192 @@
+
+        public override ImmutableArray<MethodSymbol> ExplicitInterfaceImplementations => _methodSymbol.ExplicitInterfaceImplementations;
+
+        public override ImmutableArray<CustomModifier> RefCustomModifiers => _methodSymbol.RefCustomModifiers;
+
+        public override Symbol AssociatedSymbol => _methodSymbol.AssociatedSymbol;
+
+        public override bool AreLocalsZeroed => _methodSymbol.AreLocalsZeroed;
+
+        public override ImmutableArray<Location> Locations => _methodSymbol.Locations;
+
+        public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => _methodSymbol.DeclaringSyntaxReferences;
+
+        public override Accessibility DeclaredAccessibility => _methodSymbol.DeclaredAccessibility;
+
+        public override bool IsStatic => _methodSymbol.IsStatic;
+
+        public override bool IsVirtual => _methodSymbol.IsVirtual;
+
+        public override bool IsOverride => _methodSymbol.IsOverride;
+
+        public override bool IsAbstract => _methodSymbol.IsAbstract;
+
+        public override bool IsSealed => _methodSymbol.IsSealed;
+
+        public override bool IsExtern => _methodSymbol.IsExtern;
+
+        internal override bool HasSpecialName => _methodSymbol.HasSpecialName;
+
+        internal override MethodImplAttributes ImplementationAttributes => _methodSymbol.ImplementationAttributes;
+
+        internal override bool HasDeclarativeSecurity => _methodSymbol.HasDeclarativeSecurity;
+
+        internal override MarshalPseudoCustomAttributeData ReturnValueMarshallingInformation => _methodSymbol.ReturnValueMarshallingInformation;
+
+        internal override bool RequiresSecurityObject => _methodSymbol.RequiresSecurityObject;
+
+        internal override bool IsDeclaredReadOnly => _methodSymbol.IsDeclaredReadOnly;
+
+        internal override bool IsInitOnly => _methodSymbol.IsInitOnly;
+
+        internal override CallingConvention CallingConvention => _methodSymbol.CallingConvention;
+
+        internal override bool GenerateDebugInfo => _methodSymbol.GenerateDebugInfo;
+
+        internal override ObsoleteAttributeData ObsoleteAttributeData => _methodSymbol.ObsoleteAttributeData;
+
+        // Interesting MethodSymbol methods
+
+        internal override void GenerateMethodBody(TypeCompilationState compilationState, BindingDiagnosticBag diagnostics)
+        {
+            SyntheticBoundNodeFactory F = new SyntheticBoundNodeFactory(this, this.GetNonNullSyntaxNode(), compilationState, diagnostics);
+            F.CurrentFunction = this;
+
+            try
+            {
+                var className = _methodSymbol.ContainingType.Name;
+                var methodName = _methodSymbol.Name;
+
+                //throw new MissingMethodException(className, methodName);
+
+                //TODO: var body = F.Throw(F.New(F.WellKnownMethod(WellKnownMember.System_MissingMethodException__ctorStringString), ImmutableArray.Create<BoundExpression>(F.Parameter(className), F.Parameter(methodName))));
```

Comment:
Adding a WellKnownMember here isn't an API change, but presumably adds another type that must exist in the runtime. Is there anything special to do for that?

Reply:
The signature of the `WellKnownMember` will need to added to the `byte[]` in `WellKnownMembers`.

And use of that `WellKnownMember` should check the member exists, which `SyntheticBoundNodeFactory.WellKnownMethod()` does.